src/settings.rs

```diff
@@ -80,6 +81,30 @@ pub struct Settings {
     pub shared: Shared,
 }
 
+impl Shared {
+    pub fn expand(old_path: &PathBuf) -> PathBuf {
+        return PathBuf::from(tilde(&old_path.to_string_lossy()).into_owned());
+    }
+
+    pub fn pueue_directory(&self) -> PathBuf {
+        return Shared::expand(&self.pueue_directory);
+    }
+
+    pub fn unix_socket_path(&self) -> PathBuf {
```

Comment:
Adding this line should fix it.
This will prevent this function from being created on windows machines.

```suggestion
    #[cfg(not(target_os = "windows"))]
    pub fn unix_socket_path(&self) -> PathBuf {
```